The-SMIRNOFF-force-field-format.md

```diff
@@ -235,13 +235,23 @@ The sum of formal charges for the molecule or fragment will be used to determine
 * The `partial_charge_method` attribute (default: `"CM2"`) is used to specify how uncorrected partial charges are to be generated from the quantum chemical wavefunction. Later additions will add restrained electrostatic potential fitting (RESP) capabilities.   
 
 The `<ChargeIncrement>` tags specify how the quantum chemical derived charges are to be corrected to produce the final charges.
-The `charge#increment` attribute specify how much the charge on the associated tagged atom index (replacing `#`) should be modified.
+The `chargeincrement#` attribute specify how much the charge on the associated tagged atom index (replacing `#`) should be modified.
```

Comment:
`specify` -> `specifies`

Reply:
Since there will be multiple `chargeincrementX` items, I'm changing this to "attribute**s** specify"

Reply:
Changed in d986ddf